webpack/assets/javascripts/react_app/Root/ReactApp.js

```diff
@@ -13,7 +13,8 @@ import ErrorBoundary from '../components/common/ErrorBoundary';
 import ConfirmModal from '../components/ConfirmModal';
 
 const ReactApp = ({ layout, metadata, toasts }) => {
-  const contextData = { metadata };
+  const [context, setContext] = useState({ layout, metadata });
```

Comment:
Where is `layout` used?

Reply:
I used it in the beginning and forgot to drop it, thanks!